ably/types/options.py

```diff
@@ -300,20 +311,28 @@ def __get_rest_hosts(self):
 
     def __get_realtime_hosts(self):
         if self.realtime_host is not None:
-            return self.realtime_host
-        elif self.environment is not None:
-            return f'{self.environment}-{Defaults.realtime_host}'
+            host = self.realtime_host
+        elif self.environment != "production":
```

Comment:
not sure what this is about tbh, what does `environment == "production"` mean?

Reply:
If we are not using the production environment, then the realtime host will be prefixed with the environment, `{environment}-{host}`i.e `sandbox-realtime.ably.io`. The production environment will be just the `host` without environment prefix i.e `realtime.ably.io`